node/package.json

```diff
@@ -84,14 +84,16 @@
       "aarch64-apple-darwin": "@lancedb/vectordb-darwin-arm64",
       "x86_64-unknown-linux-gnu": "@lancedb/vectordb-linux-x64-gnu",
       "aarch64-unknown-linux-gnu": "@lancedb/vectordb-linux-arm64-gnu",
-      "x86_64-pc-windows-msvc": "@lancedb/vectordb-win32-x64-msvc"
+      "x86_64-pc-windows-msvc": "@lancedb/vectordb-win32-x64-msvc",
+      "aarch64-pc-windows-msvc": "@lancedb/vectordb-win32-arm64-msvc"
     }
   },
   "optionalDependencies": {
     "@lancedb/vectordb-darwin-arm64": "0.11.1-beta.1",
     "@lancedb/vectordb-darwin-x64": "0.11.1-beta.1",
     "@lancedb/vectordb-linux-arm64-gnu": "0.11.1-beta.1",
     "@lancedb/vectordb-linux-x64-gnu": "0.11.1-beta.1",
-    "@lancedb/vectordb-win32-x64-msvc": "0.11.1-beta.1"
+    "@lancedb/vectordb-win32-x64-msvc": "0.11.1-beta.1",
+    "@lancedb/vectordb-win32-arm64-msvc": "0.11.1-beta.1"
```

Comment:
If you modify this, there are corresponding places you need to `.bumpversion.toml`:

https://github.com/lancedb/lancedb/blob/04e1f1ee4c0c56efc86f7b36e9f08f06983f1de7/.bumpversion.toml#L69-L93

Reply:
I believe it's already present in `.bumpversion.toml`. Is that wrong?https://github.com/lancedb/lancedb/pull/1770/files#diff-c157bf6a4fcea5e19ccb39979beb75b999c94a5e950492d7e423760fd4320208

Reply:
Ah I missed that. Thanks for pointing it out